test/unit/lookups/olleh_test.rb

```diff
@@ -224,8 +224,20 @@ def test_olleh_route_searching_with_waypoints
     end
   end
 
+  def test_olleh_km2_local_search
+    VCR.use_cassette('address/km2_local_search') do
+      query = Geocoder::Query.new('분당구+689', { places: 5, sr: 'RANK' })
+      lookup = Geocoder::Lookup::Olleh.new
+      result = lookup.search(query)
+      assert_equal result.first.addr, "경기도 성남시 분당구 삼평동 689"
+      assert_equal result.first.x, "965630"
+      assert_equal result.first.y, "1933900"
+      assert_equal result.first.new_addr, "경기도 성남시 분당구 판교로 335"
+    end
+  end
+
   def test_olleh_addr_step_search
-    VCR.use_cassette('address_step_search') do
+    VCR.use_cassette('address/address_step_search') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.